app/src/main/java/com/github/multimatum_team/multimatum/CalendarActivity.kt

```diff
@@ -48,6 +50,23 @@ class CalendarActivity : AppCompatActivity() {
         }
     }
 
+    /*
+    This function extract the text from the input, clear the field, add
+    a new deadline and close the keyboard.
+     */
+    private fun clearTextInput(edit: TextInputEditText, v: View) {
```

Comment:
As this function add deadline, It would be better to change it's name for a better understanding

Reply:
Good point